src/brouwers/shop/models/categories.py

```diff
@@ -25,3 +26,16 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+    def get_absolute_url(self):
+        return reverse('shop:category-detail', kwargs={'slug': self.slug})
+
+
+class CategoryCarouselImage(models.Model):
+    title = models.CharField(_('title'), max_length=100)
+    image = models.ImageField(_('category carousel image'), upload_to='shop/category/')
+    visible = models.BooleanField(_('visible'), default=True)
+
+    class Meta:
+        verbose_name = _('category carousel image')
+        verbose_name_plural = _('category carousel images')
```

Comment:
can you add a `def __str__(self)` (and decorate the model with `@python2_unicode_compatible`)?